mod/developers/views/default/admin/develop_tools/entity_explorer.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+echo '<p>' . elgg_echo('developers:entity_explorer:help') . '</p>';
+
+echo elgg_view_form('developers/entity_explorer', [
+	'action' => 'admin/develop_tools/entity_explorer',
+	'method' => 'GET',
+	'disable_security' => true,
+]);
+
+$guid = get_input('guid');
+if ($guid === null) {
+	return;
+}
+
+$show_hidden = access_show_hidden_entities(true);
+$entity = get_entity($guid);
+if (!$entity) {
+	echo elgg_echo('notfound');
+	return;
+}
+
+// Entity Information
+$entity_rows = ['type', 'subtype', 'owner_guid', 'site_guid', 'container_guid', 'access_id', 'time_created', 'time_updated', 'last_action', 'enabled'];
+
+if ($entity instanceof ElggUser) {
+	$entity_rows = array_merge($entity_rows, ['name', 'username', 'email', 'language', 'banned', 'admin', 'last_action', 'prev_last_action', 'last_login', 'prev_last_login']);	
+}
+
+if ($entity instanceof ElggGroup) {
+	$entity_rows = array_merge($entity_rows, ['name', 'description']);	
+}
+
+if ($entity instanceof ElggSite) {
+	$entity_rows = array_merge($entity_rows, ['name', 'description', 'url']);	
+}
+
+$entity_info = '<table class="elgg-table">';
+
+foreach ($entity_rows as $entity_row) {
+	$entity_info .= '<tr>';
+	$entity_info .= '<td>' . $entity_row . '</td><td>' . $entity->$entity_row . '</td>';
+	$entity_info .= '</tr>';
+}
+
+$entity_info .= '</table>';
+	
+echo elgg_view_module('inline', elgg_echo('developers:entity_explorer:info:attributes'), $entity_info);
+
+// Metadata Information
+$entity_metadata = elgg_get_metadata(['guid' => $guid, 'limit' => false]);
+
+if (empty($entity_metadata)) {
+	$metadata_info = elgg_echo('notfound');
+} else {
+	$md_columns = ['id', 'name_id', 'name', 'value_id', 'value', 'value_type', 'access_id', 'time_created', 'enabled'];
+	
+	$metadata_info = '<table class="elgg-table">';
+	$metadata_info .= '<tr>';
+	foreach ($md_columns as $md_col) {
+		$metadata_info .= '<th>' . $md_col . '</th>';
+	}
+	$metadata_info .= '<th>&nbsp;</th>';
+	$metadata_info .= '</tr>';
+	
+	foreach ($entity_metadata as $md) {
+		$metadata_info .= '<tr>';
+		foreach ($md_columns as $md_col) {
+			$metadata_info .= '<td>' . $md->$md_col . '</td>';
+		}
+		$metadata_info .= '<td>' . elgg_view('output/url', [
+			'text' => elgg_view_icon('remove'),
+			'href' => 'action/developers/entity_explorer_delete?guid=' . $entity->guid . '&type=metadata&key=' . $md->name,
+			'confirm' => true,
+		]) . '</td>';
+		$metadata_info .= '</tr>';
+	}
+	$metadata_info .= '</table>';
+}
+echo elgg_view_module('inline', elgg_echo('developers:entity_explorer:info:metadata'), $metadata_info);
+
+// Relationship Information
+$entity_relationships = get_entity_relationships($entity->guid);
+
+if (empty($entity_relationships)) {
+	$relationship_info = elgg_echo('notfound');
+} else {
+	$relationship_columns = ['id', 'time_created', 'guid_one', 'relationship', 'guid_two'];
```

Comment:
Make `guid_one` and `guid_two` links that open a new tab/ligthbox with Entity explorer for that entity